File: backend/ServiceBooking.Api/Entities/Service.cs
namespace ServiceBooking.Api.Entities;

public class Service
{
  public Guid Id { get; set; }

  public string Name { get; set; } = null!;

  public string? Description { get; set; }

  public int DurationMinutes { get; set; }

  public decimal Price { get; set; }

  public bool IsActive { get; set; }

  public DateTime CreatedAt { get; set; }

  public DateTime UpdatedAt { get; set; }

  public ICollection<Booking> Bookings { get; set; } = new List<Booking>();
}

File: frontend/types/service.ts
export interface Service {
  id: string;
  name: string;
  description?: string | null;
  durationMinutes: number;
  price: number;
  isActive: boolean;
}

export interface CreateServiceRequest {
  name: string;
  description?: string;
  durationMinutes: number;
  price: number;
}

export interface UpdateServiceRequest {
  name: string;
  description?: string;
  durationMinutes: number;
  price: number;
  isActive: boolean;
}
